lib/Models/ArcGisFeatureServerCatalogItem.js

```diff
@@ -33,6 +35,15 @@ var ArcGisFeatureServerCatalogItem = function(terria) {
    * @type {String}
    */
   this.layerDef = "1=1";
+
+  /**
+   * If set to true attempts to symbolise the data using the drawingInfo
+   * object available in the service endpoint. Defaults to false.
+   * Currently supports getting fill-color, fill-opacity, stroke-color, stroke-width
+   * from simple and uniqueVals renderers.
+   * @type {Boolean}
+   */
+  this.inheritStyleInformation = false;
```

Comment:
I think it'd be fine to default this to true, unless there's some cost to it that isn't obvious to me? And maybe `useDrawingInfo` rather than `inheritStyleInformation`?

Reply:
It sends one extra request to the overarching serving info, eg 
https://sampleserver3.arcgisonline.com/ArcGIS/rest/services/Fire/Sheep/FeatureServer/0?f=json

Reply:
Any more thoughts on this? I think `inheritStyleInformation` could be mistaken with the logic for passing configuration from groups to members.